app.js

```diff
@@ -19,7 +19,16 @@ app.get('/api/v1/projects', async (req, res) => {
     const projects = await database('projects').select();
     res.status(200).json(projects);
   } catch (error) {
-    res.status(500).json({ error });
+    res.status(404).json({ error: 'No projects found' });
```

Comment:
Should the 404 error be above inside an if statement checking the length of projects then bring the 500 error back in for the catch if there is a server error?